MediaBrowser.Api/Playback/Progressive/BaseProgressiveStreamingService.cs

```diff
@@ -362,42 +358,14 @@ private async Task<object> GetStreamResult(StreamRequest request, StreamState st
             // Use the command line args with a dummy playlist path
             var outputPath = state.OutputFilePath;
 
-            responseHeaders["Accept-Ranges"] = "none";
+            responseHeaders[HeaderNames.AcceptRanges] = "none";
 
             var contentType = state.GetMimeType(outputPath);
 
-            // TODO: The isHeadRequest is only here because ServiceStack will add Content-Length=0 to the response
```

Comment:
with this comment gone it's hard to understand why `if (isHeadRequest)` is needed... is it really needed btw? :)